src/NServiceBus.Core.Analyzer/ForwardCancellationTokenFixer.cs

```diff
@@ -34,8 +34,8 @@ public sealed override async Task RegisterCodeFixesAsync(CodeFixContext context)
             var requiredArgName = diagnostic.Properties["RequiredArgName"];
 
             var root = await context.Document.GetSyntaxRootAsync(context.CancellationToken).ConfigureAwait(false);
-            var startToken = root.FindToken(diagnostic.Location.SourceSpan.Start);
-            var invocationSyntax = startToken.Parent.AncestorsAndSelf().OfType<InvocationExpressionSyntax>().First();
+            var startToken = root.FindNode(diagnostic.Location.SourceSpan);
+            var invocationSyntax = (InvocationExpressionSyntax)startToken;
```

Comment:
Given the previous implementation always assumed there is an `InvocationExpressionSyntax` found, I left this assumption intact by doing a cast.